src/seurat/workflows/hvg_selection.nf

```diff
@@ -0,0 +1,49 @@
+nextflow.enable.dsl=2
+
+////////////////////////////////////////////////////////
+//  Import sub-workflows/processes from the utils module:
+include {
+    PUBLISH as PUBLISH_SEURAT_RDS_SCALED;
+} from '../../utils/workflows/utils.nf' params(params)
+
+////////////////////////////////////////////////////////
+//  Import sub-workflows/processes from the tool module:
+include {
+    SC__SEURAT__FIND_HIGHLY_VARIABLE_FEATURES;
+} from '../processes/feature_selection.nf' params(params)
+include {
+    SC__SEURAT__SCALING;
+} from '../processes/normalize_transform.nf' params(params)
+include {
+    GENERATE_REPORT;
+} from './create_report.nf' params(params)
+
+workflow HVG_SELECTION {
+
+    take:
+        data
+
+    main:
+        hvg = SC__SEURAT__FIND_HIGHLY_VARIABLE_FEATURES( data )
+        // TODO: REGRESS OUT
```

Comment:
Is this still planned?

Reply:
I don't know if this would be usefull to add, since it might be hard to use.
To regress something out, you would need to know the name of the metadata-column and specify this in the config file. 
Since this name depends in SCT/non-SCT + assay names, it can be hard to know which one to specify